### PR TITLE
NXP-20225: Make login screen responsive with Flexbox

### PR DESCRIPTION
**Pull request by ataillefer**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nuxeo/nuxeo/pull/197" target="_blank">ReviewNinja

File: nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp [hunk 1]
```diff
@@ -100,49 +100,129 @@ if (selectedLanguage != null) { %>
...
+  opacity: 0.8;
+  padding: 20px;
+  position: absolute;
+  width: 60%;
+}
+.welcomeText {
+  font: 12px "Lucida Grande", sans-serif;
+  text-align: left;
+  color: #454545;
+  margin: 0 0 .8em
```
Comment (lkemen) [inline nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp]: Missing ;

**Comment by lkemen** [comment]
👍

File: nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp [hunk 2]
```diff
@@ -317,145 +336,113 @@ video {
...
+          <div class="maintenanceModeMessage">
+            <div class="warnMessage">
+              <fmt:message bundle="${messages}" key="label.maintenancemode.active" /><br/>
+              <fmt:message bundle="${messages}" key="label.maintenancemode.adminLoginOnly" />
             </div>
-            <%}%>
-
+            <div class="infoMessage">
+              <fmt:message bundle="${messages}" key="label.maintenancemode.message" /> : <br/>
+              Joe is back ! Joe is back !
```
Comment (atchertchian) [inline nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp]: Joe, le taxi ?

File: nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp [hunk 3]
```diff
@@ -317,145 +336,113 @@ video {
...
+      <form method="post" action="nxstartup.faces" autocomplete="<%= fieldAutocomplete %>">
+        <!-- To prevent caching -->
+        <%
+          response.setHeader("Cache-Control", "no-cache"); // HTTP 1.1
+          response.setHeader("Pragma", "no-cache"); // HTTP 1.0
+          response.setDateHeader("Expires", -1); // Prevents caching at the proxy server
+        %>
+        <!-- ;jsessionid=<%=request.getSession().getId()%> -->
+        <!-- ImageReady Slices (login_cutted.psd) -->
+        <% if (maintenanceMode) { %>
```
Comment (atchertchian) [inline nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp]: jsession id print might be useful, but next comment could be removed

File: nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp [hunk 4]
```diff
@@ -317,145 +336,113 @@ video {
...
+        <c:if test="${param.securityError}">
+         <div class="feedbackMessage errorMessage">
+           <fmt:message bundle="${messages}" key="label.login.securityError" />
+         </div>
+        </c:if>
+        <input class="login_input" type="text" name="user_name" id="username"
+          placeholder="<fmt:message bundle="${messages}" key="label.login.username" />"/>
+        <input class="login_input" type="password" name="user_password" id="password"
+          placeholder="<fmt:message bundle="${messages}" key="label.login.password" />">
+        <% // label.login.logIn %>
```
Comment (atchertchian) [inline nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp]: same, to clean up?

**Comment by atchertchian** [comment]
:+1:

File: nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp (hunk 2 above)
Comment (ataillefer) [inline nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/login.jsp]: Right, nice catch!

**Pull request closed without merging**